ftwhttp/response_test.go

```diff
@@ -126,3 +146,28 @@ func TestResponseWithCookies(t *testing.T) {
 
 	assert.NoError(t, err)
 }
+
+func TestResponseChecksFullResponse(t *testing.T) {
+	server := testEchoServer(t)
+
+	defer server.Close()
+
+	d, err := DestinationFromString(server.URL)
+	assert.NoError(t, err)
+	req := generateRequestForTesting(true)
+
+	client, err := NewClient(NewClientConfig())
+	assert.NoError(t, err)
+	err = client.NewConnection(*d)
+
+	assert.NoError(t, err)
+
+	response, err := client.Do(*req)
+
+	assert.NoError(t, err)
+
+	assert.Contains(t, response.GetBodyAsString(), "User-Agent=[Go Tests]")
+	assert.NotContains(t, response.GetBodyAsString(), "X-Powered-By: [go-ftw]\n")
```

Comment:
Shouldn't this be the same value as on the next line, where you check the full response?

```suggestion
	assert.NotContains(t, response.GetBodyAsString(), "X-Powered-By: go-ftw")
```

Reply:
Removed this and we are checking the full response.